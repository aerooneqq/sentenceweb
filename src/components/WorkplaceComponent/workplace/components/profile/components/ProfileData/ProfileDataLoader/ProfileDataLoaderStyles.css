.profileDataLoaderContainer { 
    width: 100%;
    height: 100%;

    display: flex; 
    flex-direction: column; 

    justify-content: baseline; 
    align-items: center;
}

.profileDataAtomLoaderCont { 
    margin-top: 40px;

    width: 90%;
    height: auto; 

    display: flex;
    flex-direction: column;
}

.profileDataLoaderPropertyName { 
    width: 80px;
    height: 20px;

    background: #c0c0c0;

    border-radius: 5px;
}

.profileDataLoaderPropertyDescription { 
    margin-top: 10px;

    width: 140px;
    height: 20px;

    background: #e0e0e0;

    border-radius: 5px;
}

.profileDataLoaderPropertyInput { 
    margin-top: 10px;

    width: 100%;
    height: 30px; 

    background:  rgba(255, 45, 0, 0.5);
    
    border-radius: 5px;
}

.profileDataLoaderButton { 
    width: 160px;
    height: 35px; 

    background: rgba(255, 45, 0, 0.8);

    align-self: flex-start;

    margin-top: 40px;
    margin-left: 5%;

    border-radius: 5px;
}