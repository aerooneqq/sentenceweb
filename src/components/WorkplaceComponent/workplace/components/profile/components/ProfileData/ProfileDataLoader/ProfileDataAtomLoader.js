import React, {Component} from "react";

//Styles
import "./ProfileDataLoaderStyles.css";

export default class ProfileDataAtomLoader extends Component {
    
    constructor(props) { 
        super(props);
    }

    render() { 
        return ( 
            <div className = "profileDataAtomLoaderCont">

                <div className = "profileDataLoaderPropertyName" />
                <div className = "profileDataLoaderPropertyDescription" />
                <div className = "profileDataLoaderPropertyInput" />

            </div>
        )
    }
}