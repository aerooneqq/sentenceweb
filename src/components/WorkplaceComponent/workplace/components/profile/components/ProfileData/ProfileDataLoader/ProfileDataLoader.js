import React, {Component} from "react";

//Components
import ProfileDataAtomLoader from "./ProfileDataAtomLoader";

//Styles
import "./ProfileDataLoaderStyles.css";

export default class ProfileDataLoader extends Component { 

    constructor(props) { 
        super(props);
    }

    render() { 
        return (
            <div className = "profileDataLoaderContainer">

                <ProfileDataAtomLoader />
                <ProfileDataAtomLoader />
                <ProfileDataAtomLoader />

                <div className = "profileDataLoaderButton" />
            </div>
        )
    }
}