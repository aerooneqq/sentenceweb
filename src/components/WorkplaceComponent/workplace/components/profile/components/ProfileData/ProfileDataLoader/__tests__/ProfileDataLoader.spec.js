import React from "react";
import {shallow} from "enzyme";
import {shallowToJson} from "enzyme-to-json";
import ProfileDataLoader from "../ProfileDataLoader";
import ProfileDataAtomLoader from "../ProfileDataAtomLoader";

describe("ProfileDataLoader", () => { 
    it("should render correctly", () => { 
        expect(shallowToJson(shallow(<ProfileDataLoader />))).toMatchSnapshot();
    });

    it("should have 3 Profile data atom loaders", () => { 
        expect(shallow(<ProfileDataLoader />).find(ProfileDataAtomLoader).length).toBe(3);
    });
});