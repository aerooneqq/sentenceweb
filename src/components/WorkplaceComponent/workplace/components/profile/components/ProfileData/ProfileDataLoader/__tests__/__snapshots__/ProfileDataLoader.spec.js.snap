// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileDataLoader should render correctly 1`] = `
<div
  className="profileDataLoaderContainer"
>
  <ProfileDataAtomLoader />
  <ProfileDataAtomLoader />
  <ProfileDataAtomLoader />
  <div
    className="profileDataLoaderButton"
  />
</div>
`;
